README.md

```diff
@@ -43,6 +41,7 @@ Starting with [git commit SHA 9bd33dcc8688](https://github.com/jupyter/docker-st
 ## Other Tips
 
 * `tini -- start-notebook.sh` is the default Docker entrypoint-plus-command in every notebook stack. If you plan to modify it any way, be sure to check the *Notebook Options* section of your stack's README to understand the consequences.
+* Every notebook stack also includes an alternate script, [singleuser.sh](https://raw.githubusercontent.com/jupyter/dockerspawner/master/singleuser/singleuser.sh), which you can use as the command to run the Notebook server in single-user mode, as required by [JupyterHub](https://jupyterhub.readthedocs.org).
```

Comment:
Yes.  By using the Docker mechanism to override the run CMD, we still run under the same entrypoint.

```
docker inspect jupyter-jtyberg
...
            "Cmd": [
                "/usr/local/bin/singleuser.sh"
            ],
            "Entrypoint": [
                "tini",
                "--"
            ],
```

Perhaps I need to update the wording to make this explicit.

Now, if you change the entrypoint, you better know what you're doing.